internal/kibana/space.go

```diff
@@ -136,11 +132,14 @@ func resourceSpaceRead(ctx context.Context, d *schema.ResourceData, meta interfa
 	if diags.HasError() {
 		return diags
 	}
-	compId, diags := clients.CompositeIdFromStr(d.Id())
-	if diags.HasError() {
-		return diags
+	id := d.Id()
+	if strings.Contains(id, "/") {
```

Comment:
It's a trivial check, but maybe it makes sense to create a helper function (e.g. `IsCompositeId`) next to `func CompositeIdFromStr` to keep the details of composite ID creation in one place.